packages/components/src/__future__/Tag/_docs/Tag.stickersheet.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { TagIcon } from "~components/Icons"
+import { StickerSheet } from "~storybook/components/StickerSheet"
+import { Tag } from "../Tag"
+
+type ListProps = {
+  children: React.ReactNode
+}
+const List = ({ children }: ListProps): JSX.Element => (
+  <div className="flex flex-col gap-12 items-start">{children}</div>
+)
+
+export default {
+  title: "KAIO-staging/Tag",
```

Comment:
```suggestion
  title: "Components/Tag",
```
We can fully release this